### ISSUE TITLE
Question: Plans for adding Microdata Tags support?

### ISSUE THREAD

**Issue by HossinAmin** [tanukijs/meta-pixel#10]
I'm currently using your Nuxt Meta Pixel module in my project and finding it useful for tracking events. I was wondering about the plans for the package, specifically:

1. Would you like to add support for [Microdata Tags](https://developers.facebook.com/docs/marketing-api/catalog/guides/microdata-tags) in future versions?

2. If not, is the package's scope intended to focus solely on event tracking?

**Comment by tanukijs** [comment]
Hello @HossinAmin 👋 
Thanks for asking, let's talk about it :
1. I didn't planned it yet
2. That's what i had in mind

This feature require SSR to be effective so for this simple reason it'll never be included in `meta-pixel`.
About `nuxt-meta-pixel`, today it's up to you to do it using `useMeta` and write the right data.

I think it will not be out of scope to add a composable to `nuxt-meta-pixel` to simplify this approach. Is it what you were thinking about too ?

I'm open to suggestions of implementations too 👐

**Comment by HossinAmin** [comment]
Yes, that’s exactly how I would approach it using the `useHead` or `useMeta` composable. I don’t believe it’s out of scope either because if the project owner decides to use meta pixel as their inventory source, it’s essential to include the Miro data tags. This is necessary for the Meta Commerce Manager to detect products on the website, which is the challenge I’m currently facing in my project

**Comment by HossinAmin** [comment]
If you plan to add this feature, I’d be glad to share my implementation via a PR as a contribution to your efforts on this project

**Comment by tanukijs** [comment]
Hello @HossinAmin i didn't had the time to check this issue but i think the best solution for this would be to add typing to the  existing Nuxt `useHead`. I'll be glad to see your implementation !
